lsusb.py.in

```diff
@@ -7,6 +7,7 @@
 #
 # Copyright (c) 2009 Kurt Garloff <garloff@suse.de>
 # Copyright (c) 2013 Kurt Garloff <kurt@garloff.de>
+# Copyright (c) 2018 Kurt Garloff <kurt@garloff.de>
```

Comment:
This should read:
# Copyright (c) 2013,2018 Kurt Garloff <kurt@garloff.de>
no need to have multiple lines.

There's a whole "how to write a valid copyright line" free online tutorial on the linuxfoundation site somewhere, you should take a look at it if you are curious :)

Reply:
ok, markdown turned that # character into a huge title line, sorry about that :)